app/screens/settings-screen/security-screen.tsx

```diff
@@ -0,0 +1,196 @@
+import * as React from "react"
+import { useState } from "react"
+import { useFocusEffect } from '@react-navigation/native'
+import { Text, View } from "react-native"
+import { Button, Switch } from "react-native-elements"
+import EStyleSheet from 'react-native-extended-stylesheet'
+import RNSecureKeyStore, {ACCESSIBLE} from "react-native-secure-key-store"
+
+// Components
+import { Screen } from "../../components/screen"
+
+// Constants
+import { palette } from "../../theme/palette"
+
+// Functions
+import { translate } from "../../i18n"
+
+// Utils
+import { authenticate, isSensorAvailable } from "../../utils/biometricAuthentication"
+import { toastShow } from "../../utils/toast"
+
+// Types
+import type { ScreenType } from '../../types/screen'
+
+const styles = EStyleSheet.create({
+  container: {
+    minHeight: "100%",
+    backgroundColor: palette.white,
+    paddingLeft: 24,
+    paddingRight: 24,
+  },
+
+  settingContainer: {
+    flexDirection: "row",
+    borderBottomColor: palette.lightGrey,
+    borderBottomWidth: 1,
+  },
+
+  textContainer: {
+    marginTop: 32,
+    marginRight: 40,
+    marginBottom: 12,
+  },
+
+  title: {
+    fontSize: 20,
+    fontWeight: "bold",
+  },
+
+  subtitle: {
+    marginTop: 16,
+    fontSize: 16,
+  },
+
+  description: {
+    marginTop: 2,
+    fontSize: 14,
+    color: palette.darkGrey
+  },
+
+  switch: {
+    position: "absolute",
+    right: 0,
+    bottom: 18,
+  },
+
+  button: {
+    paddingTop: 16,
+    paddingRight: 16,
+    paddingBottom: 16,
+    paddingLeft: 0,
+    backgroundColor: palette.white,
+  },
+
+  buttonTitle: {
+    fontSize: 16,
+    color: palette.black,
+    fontWeight: "normal"
+  }
+})
+
+type Props = {
+  route: any,
+  navigation: any,
+}
+
+export const SecurityScreen: ScreenType = ({ route, navigation }: Props) => {
+
+  const { mIsBiometricsEnabled, mIsPinEnabled } = route.params
+
+  const [isBiometricsEnabled, setIsBiometricsEnabled] = useState(mIsBiometricsEnabled)
+  const [isPinEnabled, setIsPinEnabled] = useState(mIsPinEnabled)
+
+  useFocusEffect(() => {
+    RNSecureKeyStore.get("isBiometricsEnabled")
```

Comment:
Can these strings `"PIN"` and `"isBiometricsEnabled"` be extracted into `const` variables?